Store/StoreBraintreePaymentProvider.php

```diff
@@ -35,6 +35,11 @@ class StoreBraintreePaymentProvider extends StorePaymentProvider
 	 */
 	protected $site_title = '';
 
+	/**
+	 * @var string
+	 */
+	protected $device_data = null;
```

Comment:
you can just remove the `= null`, the default is null.

```suggestion
	protected $device_data;
```